paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -126,7 +126,7 @@ public class PaparazziPlugin @Inject constructor(
 
       val testInstrumentation = testVariant.instrumentation
       testInstrumentation.transformClassesWith(
-        ResourcesCompatVisitorFactory::class.java,
+        ResourcesCompatTransform::class.java,
```

Comment:
oh, sorry for the confusion.  This should remain `ResourcesCompatVisitorFactory` and the transform should remain `ResourcesCompatTransform`.